STEP_Project/tests/entitySelectTest.js

```diff
@@ -1,71 +1,29 @@
-describe('Testing selectClick function called when "select" button is clicked', ()=>
+let selectForm;
+describe('Testing the identifyEntityClass function', ()=>
 {
     let mockChrome;
-    beforeAll(function()
+    beforeEach(function()
     {
-        //initialize mock storage to mimic chrome.storage.sync
         mockChrome = 
         {
             dict: {},
-            get : function(arg1, arg2)
-            {
-                arg2(this.dict);
-            },
             set : function(pair)
             {
                 for (let key in pair)
                 {
-                    this.dict[key] = pair[key];
+                    mockChrome.dict[key] = pair[key];
                 }
-            },
-            remove : function(dataid)
-            {
-                this.dict[dataid]= undefined;
             }
         };
-
-        //initialize the UL to mimic the DOM of the settings page
-        numRows = 3;
-        tabl = document.createElement('ul');
-        tabl.setAttribute("role", "group");
-        for (let i = 0; i < numRows; i++)
-        {
-            const liObj = document.createElement('li');
-            const entry = document.createElement('div');
-            liObj.innerText = "row"+i;
-            entry.setAttribute("data-node-id", i);
-            liObj.appendChild(entry);
-            tabl.appendChild(liObj);
-            mockChrome.set({[i]:liObj.innerText})
-        }
-        document.body.appendChild(tabl);
         storageObj = mockChrome;
-    })
-
-    afterAll(function()
-    {
-        tabl.remove();
-    })
+    });
     
-    it('Should update the mock storage if entity\'s name is changed', ()=>
```

Comment:
Why are we deleting this test ?

Reply:
This was not a test for this file. It's part of another test file. Included it here initially just for syntax.